embedding/embedlite/embedthread/EmbedLiteAppThreadParent.cpp

```diff
@@ -40,7 +40,6 @@ EmbedLiteAppThreadParent::RecvInitialized()
 {
   LOGT();
   PR_SetEnv("MOZ_USE_OMTC=1");
-  PR_SetEnv("MOZ_LAYERS_PREFER_OFFSCREEN=1");
```

Comment:
That is true as currently MOZ_LAYERS_PREFER_OFFSCREEN is only red in CompositorOGL after the block containing MOZ_USE_EXTERNAL_WINDOW. However, it feel a bit odd that there would be references / indicators to an offscreen rendering even thou it is not used.